rice-middleware/impl/src/main/java/org/kuali/rice/kew/actiontaken/service/ActionTakenService.java

```diff
@@ -50,4 +51,6 @@
 
     Timestamp getLastApprovedDate(String documentId);
 
+    List<ActionTakenValue> getActionsTakenAtRouteNode(RouteNodeInstance nodeInstance);
```

Comment:
JavaDoc missing.  I know we've been bad about adding JavaDocs in the past but it would be nice not to continue that bad habit. Thanks.